pkg/log/options.go

```diff
@@ -195,4 +195,22 @@ func (o *Options) AttachCobraFlags(cmd *cobra.Command) {
 
 	// NOTE: we don't currently expose a command-line option to control ErrorOutputPaths since it
 	// seems too esoteric.
+
+	// TODO: Remove all this stuff by the 1.0 release
+	var dummy string
+	var dummy2 bool
+	cmd.PersistentFlags().StringVarP(&dummy, "v", "v", "", "deprecated")
+	cmd.PersistentFlags().StringVarP(&dummy, "stderrthreshold", "", "", "deprecated")
+	cmd.PersistentFlags().BoolVarP(&dummy2, "alsologtostderr", "", false, "deprecated")
+	cmd.PersistentFlags().BoolVarP(&dummy2, "logtostderr", "", false, "deprecated")
+
+	_ = cmd.PersistentFlags().MarkDeprecated("v", "")
```

Comment:
thanks for adding this, what does the output look like if I use it ? e.g.
`./mixs -v 2` ? do you get a warning or some other output ?

Reply:
I think it's fine if it doesn't get a warning - at least it's not crashing, and user can roll forward/backward. It's common practice in some large orgs...